target/linux/mediatek/base-files/lib/upgrade/platform.sh

```diff
@@ -18,6 +18,15 @@ platform_do_upgrade() {
 		sync
 		umount /tmp/recovery
 		;;
+	bananapi,bpi-r2)
```

Comment:
the change included inside the patch does not correlate with the patches subject/description

Reply:
Sorry, I'm not sure what do you mean - I add support for sysupgrade here, and it says so in the patch subject. Should I rename it somehow?